README.md

```diff
@@ -73,10 +54,6 @@ All content for the DIT landing pages are currently read from yaml files e.g. fo
 
 This is a public facing Ruby on Rails application that retrieves browse content from APIs and presents it. There is no underlying persistence layer, and (with the exception of the [campaign pages](#campaign-pages)) all content is retrieved from external sources.
 
-### Content for taxon pages
-
-Content for taxon pages is returned by Search API based on content_ids for world taxonomy pages and content_ids and [supergroups](https://docs.publishing.service.gov.uk/document-types/content_purpose_supergroup.html) for all other taxonomy pages.
```

Comment:
I don't think it would hurt to copy this info into the newly created taxonomy-pages.md.

Reply:
Sure, I agree it's got slightly different information in it than the separate doc. I've added a new commit to incorporate it.